xgcm/grid.py

```diff
@@ -817,9 +861,26 @@ def __init__(
             coordinates in ``ds``.
         metrics : dict, optional
             Specification of grid metrics
+        boundary : {None, 'fill', 'extend', 'extrapolate', dict}, optional
+            A flag indicating how to handle boundaries:
```

Comment:
Could docrep be used here to avoid duplication of this docstring? That's what we do elsewhere in xgcm.

Reply:
I can't figure out how to pull a single parameter out

Reply:
No worries

Reply:
Should we maybe track this as a seperate issue? I remember fighting with it for one of my PRs back in the days? I could try to refactor and write down the workflow in the near future to help other devs (and myself haha).